### DISCUSSION TITLE
retain data is not retain after shovel new broker

### DISCUSSION THREAD

**Discussion by springmvcGitHub** [rabbitmq/rabbitmq-server#7231] [Other]
hi rabbitmq expert
After I send retain data to the queue through amq.topic, and then the shovel transfers it from the queue to the new broker's amq.topic, mqtt will not retain after it picks up data from the new broker topic. How can I solve this problem?

**Comment by michaelklishin** [comment]
Retained message store in the MQTT plugin right now is limited to one node (the node the message was published on).
